rebasehelper/cli.py

```diff
@@ -167,7 +167,10 @@ def run():
             config = Config(getattr(cli, 'config-file', None))
             config.merge(cli)
             for handler in [main_handler, output_tool_handler]:
-                handler.choose_color_map()
+                if config.background == 'auto':
+                    handler.choose_color_map()
```

Comment:
This method seems redundant, why not simply pass `config.background` to `handler.set_color_map()` and handle `auto` there?